backend/apps/projects/tests.py

```diff
@@ -96,10 +244,102 @@ def test_startup_foreign_key_relation(self):
         self.assertEqual(project.startup, self.startup)
         self.assertEqual(project.startup.company_name, "Petcube")
 
-    def test_tags_field_allows_long_text(self):
+    def test_tags_field_allows_long_list(self):
         """Tags field should accept long comma-separated strings"""
-        long_tags = ",".join([f"tag{i}" for i in range(100)])
+        long_tags = [f"tag{i}" for i in range(100)]
         data = self.valid_data.copy()
         data["tags"] = long_tags
         project = Project.objects.create(**data)
         self.assertIn("tag99", project.tags)
+
+    def test_tags_field_accepts_list(self):
+        project = Project.objects.create(**self.valid_data)
+        self.assertEqual(project.tags, ['AI', 'Pets', 'IoT'])
+        self.assertIsInstance(project.tags, list)
+
+
+class ProjectAttachmentModelTest(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create(
+            email='test@example.com',
+            password='password123',
+            first_name='Test',
+            last_name='User'
+        )
+        self.startup = StartupProfile.objects.create(
+            user=self.user,
+            company_name='Test Startup',
+            email='test@test.com'
+        )
+        self.project = Project.objects.create(
+            startup=self.startup,
+            title='Test Project',
+            target_amount=Decimal('1000.00')
+        )
+
+    def test_create_attachment(self):
+        attachment = ProjectAttachment.objects.create(
+            project=self.project,
+            type='image',
+            caption='Test image',
+            order=1
+        )
+        self.assertIsInstance(attachment, ProjectAttachment)
+        self.assertEqual(attachment.project, self.project)
+        self.assertEqual(attachment.type, 'image')
```

Comment:
The test creates a ProjectAttachment without providing the required file field (FileField is not nullable/blank in the model). This will raise an IntegrityError / ValidationError and make the test (and CI) fail.

Either make the model FileField nullable/blank or update the test to provide a file using Django's SimpleUploadedFile. Preferred fix for tests:
- Add import: from django.core.files.uploadedfile import SimpleUploadedFile
- Create with a small in-memory file:
```
file = SimpleUploadedFile("test.jpg", b"filecontent", content_type="image/jpeg")
attachment = ProjectAttachment.objects.create(
    project=self.project,
    file=file,
    type='image',
    caption='Test image',
    order=1
)
```